lib/inferno/config/boot/suites.rb

```diff
@@ -25,5 +25,16 @@
     end
 
     ObjectSpace.each_object(TracePoint, &:disable)
+
+    Inferno::Entities::TestSuite.descendants.each do |descendant|
+      if descendant.id.blank?
+        raise StandardError, 'Error initializing test suites: custom test suite ID cannot be blank'
+      end
+      # When ID not assigned in custom test suites, Runnable.id will return default ID
+      # equal to the custom test suite's parent class name
+      if descendant.id == 'Inferno::Entities::TestSuite'
```

Comment:
I don't see a reason for this to be two separate checks, they should be combined into a single condition.

Reply:
Updated.  Still need to resolve rspec coverage by adding a test for the exception that is raised.